packages/forms/src/UIForm/fields/File/File.component.js

```diff
@@ -0,0 +1,125 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate } from 'react-i18next';
+import FieldTemplate from '../FieldTemplate';
+import { generateDescriptionId, generateErrorId } from '../../Message/generateId';
+
+import { I18N_DOMAIN_FORMS } from '../../../constants';
+import theme from './File.scss';
+
+class FileWidget extends React.Component {
+	constructor(props) {
+		super(props);
+		// Extract file name from value
+		if (props.value && props.value.indexOf(';name=') !== -1) {
+			const fileName = props.value.slice(props.value.indexOf(';name=') + 6, props.value.indexOf(';base64,'));
+			this.state = { fileReplaceValue: fileName };
+		} else {
+			this.state = { fileReplaceValue: '' };
+		}
+	}
+
+	loadFileContent = (onChange, event, schema) => {
+		const fileList = event.target.files;
+		// TODO manage multiple files
+		const file = fileList[0];
```

Comment:
are you sure there are file here ? (I guess we can delete the selected file)

Reply:
Yep, if user click on cancel, file could be removed.